downburst/template.xml

```diff
@@ -15,7 +15,7 @@
   <on_reboot>restart</on_reboot>
   <on_crash>restart</on_crash>
   <devices>
-    <emulator>/usr/bin/kvm</emulator>
+    <emulator>/usr/bin/qemu-kvm</emulator>
```

Comment:
@tchaikov I have doubts regarding this change, I was testing this on openSUSE, but I don't know if it will work on Ubuntu/Debian, or RedHat/CentOS

Reply:
right, on CentOS8 it should be `/usr/libexec/qemu-kvm`. see https://centos.pkgs.org/8/centos-appstream-x86_64/qemu-kvm-core-2.12.0-88.module_el8.1.0+297+df420408.3.x86_64.rpm.html .

while on ubuntu, it's `/usr/bin/kvm`.

see https://packages.ubuntu.com/eoan/amd64/qemu-kvm/filelist